site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -55,12 +55,14 @@ Not all methods in extensions APIs support promises. Sometimes that's because we
 promise support on the method yet; in many cases it's because using a promise isn't feasible for the
 method.
 
-{% if false %}
 You can check if an API method support promises by checking its API reference page:
+ 
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/AYQVtnh19vNMHoXzxZB1.png", alt="Screenshot showing a
+method that supports promises", width="800", height="280" %}
 
-{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/lDjGweNENKyqGSA2nZrp.png", alt="Screenshot of an API
-reference page with promises support noted", width="800", height="467" %}
-{% endif %}
+The screenshot above shows a method from the `chrome.tabs` API. You can see that this method
+supports promises because it displays the `Promise` pill, and also shows syntax for both the promise
+and callback forms.
```

Comment:
```suggestion
The screenshot above shows a method from the [`chrome.tabs`][api-tabs-methods] API. You can see that
this method supports promises because one of the method's signatures returns a promise. To make this
easier to see at a glance, the reference docs also display a `Promise` pill below the signatures.
```

I'm not entirely happy with this suggestion. I like that the current text is that it highlights the pill as the primary signal, as I expect that will be the easiest way for people to see promise support when skimming the docs. On the other hand, I don't think the average reader will be familiar with the "pill", so I wanted to highlight that it appears below the method signatures. Unfortunately, adding that ended up making the paragraph feel repetitive. At the moment, this suggestion is my best effort to blend those concerns. 

Also, I figured that since we're calling out tabs API methods, it made sense to link to them as well. This will let users jump straight to the docs to see non-screenshot examples if they wish. (I can't suggest to the end of the document because nothing changed downt here.)

```
[api-tabs-methods]: /docs/extensions/reference/tabs#method-captureVisibleTab
```